### DISCUSSION TITLE
Trouble with G6 session start failed

### DISCUSSION THREAD

**Discussion by RayDubbs** [Navid200/xDrip#74] [Dexcom]
![Uploading Screenshot_20211008-182924_xDrip+.jpg…]()
Hello everyone,  I'm using a g6 with android.   I had my sensor out for a few days while waiting for a new sensor.   Put in sensor this morning and started getting "session start failed: clock not syncronized or other error" My phone also did a software update while I was waiting for my new sensor.  Here is a screen shot. 
![Uploading Screenshot_20211008-182924_xDrip+.jpg…]()

**Comment by RayDubbs** [comment]
![Uploading Screenshot_20211008-182924_xDrip+.jpg…]()

**Reply by Navid200** [reply]
Are you working on a mobile or a desktop?
If a desktop, all you need to do to upload is to tap on the bottom border of the frame you are typing in and it will let you chose an image.

**Reply by RayDubbs** [reply]
Mobile, not sure why it isn't working.  I'll try again. 
![Screenshot_20211008-182924_xDrip+](https://user-images.githubusercontent.com/81883330/136636134-cb32307d-d215-40cf-a306-67754a2aa2c6.jpg)

**Reply by RayDubbs** [reply]
@Navid200

**Reply by Navid200** [reply]
Can you  upload the system status page?

**Reply by RayDubbs** [reply]
@Navid200 
![Screenshot_20211008-203212_xDrip+](https://user-images.githubusercontent.com/81883330/136637373-9ebc4e66-0b94-49b4-8ec1-e48594e738b4.jpg)

**Reply by Navid200** [reply]
That's good.  On that same page, please swipe left so that you go to the G5/G6 status page and show that one too.

**Reply by RayDubbs** [reply]
![Screenshot_20211008-203859_xDrip+](https://user-images.githubusercontent.com/81883330/136637612-52829f3c-36e9-4610-937b-aa77beabb796.jpg)

**Reply by Navid200** [reply]
The problem is the number of days.
With that transmitter, you can start a sensor as long as the "Transmitter Days" shown on that page is less than 100.

**Reply by RayDubbs** [reply]
Is there a work around or do I need a new transmitter?

**Reply by Navid200** [reply]
You need a new transmitter.  But, I wouldn't throw it away.

**Reply by RayDubbs** [reply]
Thank you for your help.
